sycl/unittests/thread_safety/InteropKernelEnqueue.cpp

```diff
@@ -1,65 +0,0 @@
-//==-------- InteropKernelEnqueue.cpp --- Thread safety unit tests ---------==//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-
-#include <cstddef>
-#include <gtest/gtest.h>
-#include <helpers/KernelInteropCommon.hpp>
-#include <helpers/PiMock.hpp>
-#include <sycl/sycl.hpp>
-
-#include "ThreadUtils.h"
-
-namespace {
-using namespace sycl;
-
-constexpr std::size_t NArgs = 16;
-constexpr std::size_t ThreadCount = 4;
-constexpr std::size_t LaunchCount = 8;
-
-pi_uint32 LastArgSet = -1;
-std::size_t LastThread = -1;
-pi_result redefined_piKernelSetArg(pi_kernel kernel, pi_uint32 arg_index,
-                                   size_t arg_size, const void *arg_value) {
-  EXPECT_EQ((LastArgSet + 1) % NArgs, arg_index);
-  LastArgSet = arg_index;
-  std::size_t ArgValue = *static_cast<const std::size_t *>(arg_value);
-  if (arg_index == 0)
-    LastThread = ArgValue;
-  else
-    EXPECT_EQ(LastThread, ArgValue);
-  return PI_SUCCESS;
-}
-
-TEST(KernelEnqueue, InteropKernel) {
```

Comment:
This [comment](https://github.com/intel/llvm/pull/12172/files#r1481365200) gives more context on what happened to this kind of tests.

This test has moved to [set_arg_interop](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/DeprecatedFeatures/set_arg_interop.cpp) e2e test.